website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When multiple selections are possible.
+- When list items are displayed in groups.
+- When you require complex structured content to be displayed within the list (and trigger).
+
+### When not to use
+
+- To display a list of links (or buttons) in a menu, consider the [HDS Dropdown](/components/dropdown) component instead.
+- When a simple select is needed for a form, consider the [HDS Select](/components/form/select) form element.
+- As a means to filter an array of objects within a data set; e.g., when presented in a tabular format or list. Instead, consider other [filtering patterns](/patterns/filter-patterns).
+
+## Components
+
+### Single
+
+Selection is limited to a single list item with structured content.
```

Comment:
@majedelass Maybe remove "with structured content" since to me it implies that it has to contain structured content?

Also I think it would be clearer to refer to "option" or "option" item vs. "list item" (to match terminology for HTML select and option elements and to make it clearer that users are selecting among options)

Reply:
Let me know if you disagree about option vs. "list item" btw, maybe I'm overthinking it...

Reply:
We changed to this so that we encourage folks to use structured content - using simple text strings then should use the select component.

I do agree with your recommendation to change list item to option to be consistent:

```suggestion
Selection is limited to a single option item with structured content.
```

Reply:
Will make this update locally.